src/desktop/apps/fairs/helpers/view_helpers.coffee

```diff
@@ -52,13 +53,21 @@ module.exports =
     fair.is_published and fair.profile?.is_published and fair.profile?.icon?.url
 
   isNotOver: (fair) ->
-    Date.parse(fair.end_at) > new Date
+    endOfFair = moment.utc(fair.end_at)
+    offset = endOfFair.utcOffset() 
+    endOfFair = endOfFair.endOf("day")
+    now = moment().utcOffset(offset)
+    now.diff(endOfFair) < 0
 
   isPast: (fair) ->
     @isEligible(fair) and @isOver(fair)
 
   isOver: (fair) ->
-    Date.parse(fair.end_at) < new Date
+    endOfFair = moment.utc(fair.end_at)
+    offset = endOfFair.utcOffset() 
+    endOfFair = endOfFair.endOf("day")
+    now = moment().utcOffset(offset)
+    now.diff(endOfFair) > 0
```

Comment:
As mentioned on Slack, when comparing times with moment.js, _I think_ it will handle the timezone for us so we don't need to convert. we can probably just do
```coffeescript
isOver: (fair) ->
  endOfFair = moment(fair.end_at).endOf("day") # assuming we want the end of day in the UTC timezone
  moment().isAfter(endOfFair)
```

I have to look up moment.js doc every time I manipulate the timezone and it's not very straightforward. A few things I noticed in your implementation:

```coffeescript
endOfFair = moment.utc(fair.end_at)
offset = endOfFair.utcOffset()
```
`endOfFair` is switched to utc mode, so `offset` here will always be `0`.

```coffeescript
now = moment().utcOffset(offset)
```
If the intention here is to convert now to utc timezone, we can simply do `moment().utc()`

Reply:
I used the logic provided in this blogpost: https://maggiepint.com/2016/05/14/moment-js-shows-the-wrong-date/
 
```

12 | moment.utc('2016-05-14T08:33:14-03:00').format()&quot;2016-05-14T11:33:14Z&quot;
-- | --

```

I was getting the wrong offset when using the regular `moment()` method.